xmppserver/src/main/java/org/jivesoftware/openfire/muc/MultiUserChatManager.java

```diff
@@ -592,8 +611,8 @@ public void refreshService(String subdomain) {
             pstmt.setString(1, subdomain);
             rs = pstmt.executeQuery();
             if (rs.next()) {
-                String description = rs.getString(1);
-                Boolean isHidden = Boolean.valueOf(rs.getString(2));
+                String description = rs.getString(2);
```

Comment:
Am I right that `LOAD_SERVICE` is only used in this code block and so I am wondering why `serviceID` was added to the query when not used here?  Just for consistency?

Reply:
Indeed, I did that for consistency. Querying for a column that isn't used (yet?) seemed about as silly as having an inconsistent column definition between the `LOAD_SERVICE` AND `LOAD_SERVICES` queries. 

I'm very open to change this around if anyone can make even a mediocre argument for such a change.